packages/swift/build.sh

```diff
@@ -103,28 +102,26 @@ termux_step_host_build() {
 termux_step_pre_configure() {
 	export SWIFT_ARCH=$TERMUX_ARCH
 	test $SWIFT_ARCH == 'arm' && SWIFT_ARCH='armv7'
```

Comment:
Create a `termux_step_configure` step for this Swift package and move these two lines and the clang headers lines above there, so these environment variables are defined later for the Swift build.